Makefile

```diff
@@ -169,7 +180,7 @@ httpproxylet:
 nginxTest:
 	cd testNginx && docker build --tag ${IMAGE_REPO}/testnginx:${IMAGE_TAG} .
 
-allimages: baseimage testimage cloudimage clientimage echoproxylet simpleauth debugcloudimage httpproxy httpproxylet
+allimages: baseimage testimage cloudimage clientimage echoproxylet simpleauth debugcloudimage httpproxy httpproxylet nginxTest
```

Comment:
The recently added `nginxTest` image had not been appended to this target yet which made the target name a bit of a liar.